prov/verbs/src/verbs_mr.c

```diff
@@ -201,6 +201,11 @@ vrb_mr_ofi2ibv_access(uint64_t ofi_access, struct vrb_domain *domain)
 			      IBV_ACCESS_REMOTE_WRITE |
 			      IBV_ACCESS_REMOTE_ATOMIC;
 
+	/* No strict ordering requested */
+	if (domain->util_domain.tx_msg_order == FI_ORDER_NONE &&
+	    domain->util_domain.rx_msg_order == FI_ORDER_NONE)
```

Comment:
FI_ORDER_NONE is 0, which isn't particularly useful as a flag definition.  Can you change these checks to !tx_msg_order && !rx_msg_order?  (I remove FI_ORDER_NONE from the header files in v2 because it's a flag of 0.)

Of course, this change depends on how we handle the comments from the first patch.